docs/products/postgresql/concepts/high-availability.md

```diff
@@ -30,14 +30,18 @@ Aiven's Business and Premium plans offer
 
 ## Failure handling
 
-**Minor failures**, such as service process crashes or temporary loss of
+### Minor failures
+
+Minor failures, such as service process crashes or temporary loss of
 network access, are handled automatically by Aiven in all plans without
 any major changes to the service deployment. The service automatically
-restores normal operation once the crashed process is automatically
```

Comment:
I think we should keep the automatic word here. Automatic fail-overs is a differentiator feature. There is also manual fail-overs.